server/schedule/placement/rule_manager.go

```diff
@@ -365,3 +367,14 @@ func (m *RuleManager) Batch(todo []RuleOp) error {
 	log.Info("placement rules updated", zap.String("batch", fmt.Sprint(todo)))
 	return nil
 }
+
+func (m *RuleManager) rebuildRuleList() (ruleList, error) {
+	for _, r := range m.rules {
+		r.group = m.groups[r.GroupID]
```

Comment:
When will the `RuleManager.Group` (i.e `m.groups`) be initialized?

Reply:
RuleManager.Initialize(): load from etcd // not included in this PR

Reply:
Ok.